source/System.h

```diff
@@ -173,6 +174,10 @@ class System {
 	// The smallest arrival period of a fleet (or 0 if no fleets arrive)
 	int MinimumFleetPeriod() const;
 
+	const std::vector<RaidFleet> &RaidFleets() const;
+	// Whether or not raids are allowed to spawn in this system.
+	bool NoRaids() const;
```

Comment:
You could also fold this into `RaidFleets()` so that it returns an empty vector if `"no raids"` is used.

Reply:
I like the idea but should I keep an empty vector in the empty namespace or smth then?

Reply:
local static https://github.com/endless-sky/endless-sky/blob/master/source/MaskManager.cpp#L98